nest-forms-backend/src/utils/files.ts

```diff
@@ -1,5 +1,27 @@
+import * as jwt from 'jsonwebtoken'
+
+import { FormWithFiles } from './types/prisma'
+
 // PDF_FORM_FAKE_FILE_ID is used to tell GinisService functions that they shouldn't look for this file in database
 export const PDF_FORM_FAKE_FILE_ID = 'pdf-form-fake-id'
 export const fileIdIsValid = (fileId: string): boolean =>
   fileId !== PDF_FORM_FAKE_FILE_ID
 export const PDF_EXPORT_FILE_NAME = 'pdf-export.pdf'
+
+export const getFileIdsToUrlMap = (
+  form: FormWithFiles,
+  jwtSecret: string,
+  selfUrl: string,
+): Record<string, { url: string; fileName: string }> => {
```

Comment:
Also one more thing: we can use the type `RenderSummaryEmailFileIdInfoMap` here, however since it is not tied only to the emails now, we can rename it to maybe just `FileIdInfoMap`. WDYT?